src/tools/gophish_export.py

```diff
@@ -34,21 +34,20 @@
 # cisagov Libraries
 from tools.connect import connect_api
 
-args = docopt(__doc__, version="v2.0")
-# Support Insecure Request waring.
-
+from ._version import __version__
 
+# Support Insecure Request warning.
```

Comment:
These comments should really say `# Disable "Insecure Request" warning` and should include a short explanation of why that is necessary.

Reply:
In looking at this error more, I realized this can be disabled for all the `gophish-*` scripts at a higher level within `src/util/connect.py`. This will require more changes than just comments so I will make this change in a separate PR. 

I also discovered that the scope can be narrowed by adding `urllib3.exceptions.InsecureRequestWarning` within `urllib3.disable_warnings()`.  Additional testing will be required again, holding off on this for a separate PR. 

I have added the comment to all scripts for the time being to ensure clarity early on.

Reply:
Sounds good - thanks!